bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -120,6 +123,46 @@
 		</config-description>
 	</thing-type>
 
+	<!-- Ohmpilot Thing Type -->
+	<thing-type id="ohmpilot">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Fronius Ohmpilot</label>
+		<description>Fronius Ohmpilot</description>
+
+		<channels>
+			<channel id="ohmpilotpowerrealsum" typeId="ohmpilot_powerreal">
+				<label>Real Power Sum</label>
+			</channel>
+			<channel id="ohmpilotenergyrealsumconsumed" typeId="ohmpilot_energy">
+				<label>Real Energy Consumed</label>
+			</channel>
+			<channel id="ohmpilottemperaturechannel1" typeId="ohmpilot_temperature">
+				<label>Temperature</label>
+			</channel>
+			<channel id="ohmpilotstatecode" typeId="ohmpilot_state">
+				<label>State</label>
+			</channel>
+			<channel id="ohmpiloterrorcode" typeId="devicestatus_errorcode">
+				<label>Error Code</label>
+			</channel>
+		</channels>
+
+		<properties>
+			<property name="model"/>
+			<property name="serial"/>
+		</properties>
```

Comment:
This is not needed (as no value is set), these properties will be set by your code.
You can remove them also from the thing type `meter`.